BaseFilter/sections/antiadblock.txt

```diff
@@ -2546,10 +2546,8 @@ videosgay.me#%#//scriptlet('set-constant', 'cRAds', 'true')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/163449
 perchance.org#%#//scriptlet('set-constant', 'ad1sAreShowing', 'trueFunc')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/163466
-smsonline.cloud#@##ad-slot
-smsonline.cloud#@#ins.adsbygoogle[data-ad-client]
-smsonline.cloud#@#ins.adsbygoogle[data-ad-slot]
-@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=smsonline.cloud
+smsonline.cloud#%#(()=>{const e=()=>{document.querySelectorAll(".chakra-portal").forEach((e=>{e.querySelector('.chakra-modal__overlay[style*="opacity"]')&&e.setAttribute("style","display: none !important;")}))},t=()=>{},a=function(t,a){const r={name:t,listener:a};requestAnimationFrame((()=>{try{"rewardedSlotGranted"===r.name&&setTimeout(e,2e3),r.listener()}catch(e){}}))};window.googletag={cmd:[],pubads:()=>({addEventListener:a,removeEventListener:t,refresh:t,getTargeting:()=>[],setTargeting:t,disableInitialLoad:t,enableSingleRequest:t,collapseEmptyDivs:t,getSlots:t}),defineSlot:()=>({addService(){}}),defineOutOfPageSlot:t,enableServices:t,display:t,enums:{OutOfPageFormat:{REWARDED:1}}},googletag.cmd.push=e=>{try{e()}catch(e){}return 1}})();
+@@||smsonline.cloud^$genericblock

```

Comment:
I think that this rule is required only for platforms which do not support JS rules or cannot update/download new, so maybe it would be better to add a hint?
For example:
```adblock
!+ NOT_PLATFORM(windows, mac, android, ext_chromium, ext_opera)
@@||smsonline.cloud^$genericblock
```